### ISSUE TITLE
Ghost death flash is in wrong spot when invisible. 

### ISSUE THREAD

**Issue by SpookySquidGames** [binji/porklike.gb#6]
Killing ghosts when they're invisible (with or without a weapon) will cause their death sprite flash to occur in their location from the previous turn.

**Comment by binji** [comment]
Good catch! I actually had a TODO in the code about this, but forgot to fix it. This should be fixed in v1.0.9.

BTW, since you seem to have been playing this for a while, I'm curious if you have any thoughts about features as well as bugs?

**Comment by SpookySquidGames** [comment]
Great! Had a feeling that might be a known issue. 

Hard to say what features I'd like to see, it's a such a perfect little game. Thinking on it though, I've played it a bunch on both gameboy and the original pico-8 version and I have never actually finished it. I've made it to floor 9 but never past that. So having a more casual option that starts you with more hearts might be nice.  

If adding save functionality is possible a stats screen would be a nice addition. Something that lets me know how many of each monster I've defeated (which might give a hint that the worstlord that chases you can be killed, though that was fun to discover for myself). Furthest floor reached, what difficulties I've beaten, number of wins etc. Could even be something more advanced similar to the book in Spelunky. 

More weapons, levels, enemies would be great but I'd definitely be worried about throwing off the balance the game has achieved. I wonder if Krystian has ideas that ended up on the cutting room floor due to time? If there is additional game content added it would be nice to have it as a separate option. Eg. pick between Porklike Classic and Porklike Extended at the title screen.
